README.md

```diff
@@ -218,14 +218,15 @@ You can experiment with the class fields proposal using the following complete i
 - Public fields are [enabled by default](https://www.chromestatus.com/feature/6001727933251584) in Chrome 72 / V8 7.2
 - Private fields are [enabled by default](https://www.chromestatus.com/feature/6035156464828416) in Chrome 74 / V8 7.4
 - Public instance fields are [enabled by default](https://bugzilla.mozilla.org/show_bug.cgi?id=1499448) in Firefox 69
+- Public static fields are [enabled](https://bugzilla.mozilla.org/show_bug.cgi?id=1535804) in Firefox 75
 - [Moddable XS](https://blog.moddable.com/blog/secureprivate/)
 - [QuickJS](https://www.freelists.org/post/quickjs-devel/New-release,82)
 - [TypeScript 3.8](https://devblogs.microsoft.com/typescript/announcing-typescript-3-8/#ecmascript-private-fields)
 
 Further implementations are on the way:
 
 - [Out for review](https://bugs.webkit.org/show_bug.cgi?id=174212) in Safari/JSC
-- [In progress](https://bugzilla.mozilla.org/show_bug.cgi?id=1499448) in Firefox/SpiderMonkey
+- [Private instance fields](https://bugzilla.mozilla.org/show_bug.cgi?id=1562054) and [private instance methods and accessors](https://bugzilla.mozilla.org/show_bug.cgi?id=1435826) are in progress in Firefox/SpiderMonkey
```

Comment:
I'm not sure how accurate "in progress" is; it might be more like a tracking issue at this point.